exporter/trace/trace_proto.go

```diff
@@ -84,7 +84,7 @@ func protoFromSpanData(s *export.SpanData, projectID string) *tracepb.Span {
 		SameProcessAsParentSpan: &wrapperspb.BoolValue{Value: !s.HasRemoteParent},
 	}
 	if s.ParentSpanID != s.SpanContext.SpanID && s.ParentSpanID.IsValid() {
-		sp.ParentSpanId = fmt.Sprintf("%.16x", s.ParentSpanID)
```

Comment:
This line was originally introduced in https://github.com/open-telemetry/opentelemetry-go/pull/160 and then was moved to this repo.

@ymotongpoo was this line imported from OpenCensus? Do OpenCensus and OpenTelemetry have different string representation of SpanIDs?